src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/elbow/RotateToAngleRadians.java

```diff
@@ -31,17 +27,23 @@ public RotateToAngleRadians(double angle){
     @Override
     public void initialize() {
         super.initialize();
-        pidController = new ProfiledPIDController(PID.getKp(), PID.getKi(), PID.getKd(), CONSTRAINTS);
+        if(RobotMap.ROBOT_TYPE != Robot.RobotType.SIMULATION) {
+            pidController = new ProfiledPIDController(PID.getKp(), PID.getKi(), PID.getKd(), CONSTRAINTS);
+        }else{
+            pidController = new ProfiledPIDController(SIM_PID.getKp(), SIM_PID.getKi(), SIM_PID.getKd(), CONSTRAINTS);
+        }
         legalGoalAngle = elbow.getLegalGoalAngle(wantedAngle);
         pidController.reset(new TrapezoidProfile.State(elbow.getAngleRadians(), elbow.getVelocity()));
+
+        elbow.setGoalAngle(legalGoalAngle);
```

Comment:
An honest question: Why is this change a part of a PR about simulation?

Reply:
cuz i added a dashboard put and prob didnt delete the empty line